optree/ops.py

```diff
@@ -127,2456 +123,374 @@
 """Literal constant that treats :data:`None` as a pytree leaf node."""
 
 
-def tree_flatten(
-    tree: PyTree[T],
-    /,
-    is_leaf: Callable[[T], bool] | None = None,
-    *,
-    none_is_leaf: bool = False,
-    namespace: str = '',
-) -> tuple[list[T], PyTreeSpec]:
-    """Flatten a pytree.
+tree_flatten = tree.flatten
```

Comment:
I'd like to keep the function definitions in `optree.ops` and re-export them to `optree.tree`.

Reply:
I thought about it, but I finally decide to implement the proposal in current approach. Major reason is the documentation issue, I want to make sure doc string in `optree.tree` is the ground truth.